library/std/src/sys/sgx/abi/mem.rs

```diff
@@ -12,6 +12,20 @@ pub(crate) unsafe fn rel_ptr_mut<T>(offset: u64) -> *mut T {
 
 extern "C" {
     static ENCLAVE_SIZE: usize;
+    static HEAP_BASE: u64;
+    static HEAP_SIZE: usize;
+}
+
+/// Returns the base memory address of the heap
+#[unstable(feature = "sgx_platform", issue = "56975")]
+pub fn heap_base() -> *const u8 {
```

Comment:
Doesn't need to be `pub` (and therefore doesn't need a stability attribute), same for `heap_size`